code/game/objects/items/food/egg.dm

```diff
@@ -85,6 +85,24 @@
 	else
 		..()
 
+/obj/item/food/egg/afterattack_secondary(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
```

Comment:
You may see fit to respect the parent return of `afterattack_secondary`. 
Something like (`if(. == SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN); return`